static/src/stylesheets/module/content/_media-player.scss

```diff
@@ -20,9 +20,13 @@ $ima-controls-height: 70px;
     -webkit-transform: translateZ(0); // fixes iOS hover bug
 }
 .gu-media-wrapper--audio {
-    background-color: $vjs-control-colour;
+    background-color: rgba($vjs-control-colour, .8);
```

Comment:
Good spot, shouldn't have done that have reverted it to $vjs-control-colour; Which are the others you are referring to?